Documentation/subcommands/version.md

```diff
@@ -6,7 +6,7 @@ This command prints the rkt version, the appc version rkt is built against, and
 
 ```
 $ rkt version
-rkt Version: 1.9.1
+rkt Version: 1.10.0
```

Comment:
According to glide, we use appc version 0.8.5:

```
- package: github.com/appc/spec
  version: v0.8.5
```

We still have other mentions:
```
$ git grep -- '0\.7\.4'|wc -l
27
```